solveCSS not being supported.css

```diff
@@ -0,0 +1,113 @@
+/* All browsers behave differently and have their own type of CSS. In the case of Chrome browsers, devs need to set the WebKit pixel ratio. The code below demonstrates how to do so with various Chrome versions. */
```

Comment:
I think Safari is not supported in this css file.

Reply:
Really?